linux_pipeline/Jenkinsfile_upstream_kernel

```diff
@@ -573,5 +606,26 @@ pipeline {
         }
       }
     }
+
+    stage('publish_result_todb') {
```

Comment:
this stage name is too long and is confusing when it appears in blue ocean, try to shorten it to appear on one line. publish_results can be a good choice